File: 2015 ICPC North American Qualifier Contest/simonsays.pas
program SimonSays;

var
  times, i: integer;
  s: string;

begin
  readln(times);
  
  for i := 1 to times do
  begin
    readln(s);
    
    if Pos('Simon says', s) = 1 then  // Check if it starts at position 1
    begin
      Delete(s, 1, 11);  // Delete first 11 characters
      writeln(s);
    end;
  end;
end.

File: 2015 ICPC North American Qualifier Contest/simonsays.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

int main() {
    int times;
    scanf("%d", &times);
    getchar(); // consume newline
    
    for(int i = 0; i < times; i++) {
        char s[256];
        fgets(s, sizeof(s), stdin);
        
        // Remove newline character
        s[strcspn(s, "\n")] = 0;
        
        if(strstr(s, "Simon says") == s) { // Check if it starts with "Simon says"
            char *result = s + 11; // Point to after "Simon says"
            printf("%s\n", result);
        }
    }
    
    return 0;
}